litmus-portal/frontend/src/components/Sections/Settings/TeammingTab/Invitation/Sent/index.tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable camelcase */
+import {
+  Avatar,
+  Table,
+  TableCell,
+  TableContainer,
+  TableRow,
+} from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import config from '../../../../../../config';
+import { RootState } from '../../../../../../redux/reducers';
+import userAvatar from '../../../../../../utils/user';
+import ButtonFilled from '../../../../../Button/ButtonFilled';
+import ButtonOutline from '../../../../../Button/ButtonOutline';
+import useStyles from './styles';
+
+interface UserInvite {
+  _id: string; // eslint-disable-line no-eval
+  username: string;
+  email: string;
+  name: string;
+  logged_in: boolean; // eslint-disable-line no-eval
+  created_at: string; // eslint-disable-line no-eval
+  updated_at: string; // eslint-disable-line no-eval
```

Comment:
no eslint-dsiable are required here too...these has been added with recent changes to master